service-worker.js

```diff
@@ -0,0 +1,23 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = '{{ site.time }}';
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
+toolbox.cache.maxEntries = 150;
+toolbox.cache.maxAgeSeconds = 604800;
```

Comment:
I don't think so.

 ```js
const VERSION = '{{ site.time }}';
```

That ensure the cache to be invalided after a deployement of the website.

A stale website in cache won't show up but will probably be cached a week.